generals/game.py

```diff
@@ -1,14 +1,18 @@
 import warnings
 import numpy as np
 import gymnasium as gym
-from typing import Dict, List
+from typing_extensions import TypeAlias
+
 from generals.config import DIRECTIONS, PASSABLE, MOUNTAIN, INCREMENT_RATE
 
 from scipy.ndimage import maximum_filter
 
+Observation: TypeAlias = dict[str, np.ndarray]
+Info: TypeAlias = dict[str, int]
```

Comment:
probably `dict[str, any]` is better, now `int` is just a coincidence

Reply:
done